### DISCUSSION TITLE
Daikin FDYAN140AV1 ducted wifi?

### DISCUSSION THREAD

**Discussion by cstaylor80** [revk/xESP32-Faikout#551] [General]
Hey everyone,

I’m looking into using the ESP32-Faikin project to control my ducted Daikin air conditioner, but I’m not sure if it will be compatible with my setup. My system has these models:

Indoor Unit: FDYAN140AV1
Outdoor Unit: RZA140C2V1
PCB Board: PC14010
Has anyone tried this project with similar models? Will it work out of the box, or will I need to make any changes? Also, does the board support communication via the S21 port OR any other port?????

Attached are some pics of the board as best I could.

Cheers!
![20241115_090736](https://github.com/user-attachments/assets/b8c7f17f-2219-4d26-a1d5-5e6b8a719721)
![20241115_090743](https://github.com/user-attachments/assets/69dce5fa-a459-4dd9-8911-31b4deb8faa2)
![20241115_090415](https://github.com/user-attachments/assets/78bfd925-9db9-4d6b-a475-0644ed23c0bf)

**Comment by revk** [comment]
I think that has an X50A, so you may be in luck. There are some instructions for connecting to an X50A and I think X35A for power.

**Reply by cstaylor80** [reply]
Hi Yes It does have an X50A I will check if it has a x35a also, is there a wire diagram?

**Reply by revk** [reply]
Link on the bottom of the wiring page has some.

https://github.com/revk/ESP32-Faikin/wiki/Wiring

**Comment by sbwilson** [comment]
Did you get this to work? I have a similar model (FDYAN125AV1) and was wondering if it was worth buying a ladder to get into the ceiling to check...

**Comment by f616** [comment]
Hi,

I have two FDXM25F9 and one FDXM35F9 integrated with FAIKIN and HA.
The FAIKIN is connected to the X50A and X35A (for powering the FAIKIN).

Don't know if it's similar to yours or not, but I'm sharing a picture of the board and the ports used.

![image](https://github.com/user-attachments/assets/9abe5ae3-ab29-4ea0-8bb6-70dfc1f583ce)

I got the cables from [Aliexpress](https://www.aliexpress.com/item/1005005465304996.html)

**Reply by matt2828** [reply]
Hi did you get it to wok, i have the same connection hooked up witht the genuine faikin but no serial data?
